utils/channel_utils.jsx

```diff
@@ -47,3 +48,7 @@ export function findNextUnreadChannelId(curChannelId, allChannelIds, unreadChann
 
     return -1;
 }
+
+export function getRedirectChannelNameForTeam(teamId) {
+    return getRedirectChannelNameForTeamRedux(store.getState(), teamId);
```

Comment:
The point of doing this here is because we are using a redux selector and passing the state to it, we normally do this kind of things inside this utils modules, not allover the application.

Moving it into a selector have the same problem, to use a new selector, I can use the mattermost-redux selector itself. The unique point to have this function here is to have the `store.getState()` in the scope.

Reply:
Whoops. I had a comment I wanted to make here, and I forgot to submit it apparently.

I think this is the opposite of what we want to encourage though. Outside of the `LocalStorageStore`, we're usually using `getRedirectChannelNameForTeam` somewhere that we have the store already. In a few of those places, the store is imported directly, but most of the time that this is called seems to be in either `mapStateToProps` or in an action.

I'd be fine merging this, but I think we need to be careful to avoid doing this too often.